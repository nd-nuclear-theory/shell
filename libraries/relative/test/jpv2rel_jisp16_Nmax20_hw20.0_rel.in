20 4
Vrel_JISP16_bare_Jmax4.hw20
jisp16_Nmax20_hw20.0_rel.dat
