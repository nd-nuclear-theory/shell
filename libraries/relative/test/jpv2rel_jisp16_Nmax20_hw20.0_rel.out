Operator setup...
  Truncation: Nmax 20 Jmax 4
  Matrix elements: 1285
  Allocated: 2035
Reading relative operator file (JPV format)...
  Filename: Vrel_JISP16_bare_Jmax4.hw20
  Input sector (raw labels): J 0 S 0 L 0 Lp 0 ipcut 61 dimension 120 mn 938.92 hw 20 ident 0
    After transposition to upper triangle: (Lp,L)=(0,0) flip 0
    Input sector properties: expected m.e. 7260 nmax 119 Nmax_bra 238 Nmax_ket 238
    Subspace labels: bra [ 0 0 0 1 0 ] (dimension 11) ket [ 0 0 0 1 0 ]  (dimension 11)
    Sector storage: sector_index 0 subspace_index_bra 0 subspace_index_ket 0 entries 132
  Input sector (raw labels): J 1 S 1 L 0 Lp 0 ipcut 61 dimension 120 mn 938.92 hw 20 ident 0
    After transposition to upper triangle: (Lp,L)=(0,0) flip 0
    Input sector properties: expected m.e. 7260 nmax 119 Nmax_bra 238 Nmax_ket 238
    Subspace labels: bra [ 0 1 1 0 0 ] (dimension 11) ket [ 0 1 1 0 0 ]  (dimension 11)
    Sector storage: sector_index 1 subspace_index_bra 1 subspace_index_ket 1 entries 132
  Input sector (raw labels): J 1 S 1 L 0 Lp 2 ipcut 61 dimension 120 mn 938.92 hw 20 ident 0
    After transposition to upper triangle: (Lp,L)=(0,2) flip 1
    Input sector properties: expected m.e. 7260 nmax 119 Nmax_bra 238 Nmax_ket 240
    Subspace labels: bra [ 0 1 1 0 0 ] (dimension 11) ket [ 2 1 1 0 0 ]  (dimension 10)
    Sector storage: sector_index 2 subspace_index_bra 1 subspace_index_ket 7 entries 110
  Input sector (raw labels): J 1 S 1 L 2 Lp 0 ipcut 61 dimension 120 mn 938.92 hw 20 ident 0
    After transposition to upper triangle: (Lp,L)=(0,2) flip 0
    Input sector properties: expected m.e. 7260 nmax 119 Nmax_bra 238 Nmax_ket 240
    Subspace labels: bra [ 0 1 1 0 0 ] (dimension 11) ket [ 2 1 1 0 0 ]  (dimension 10)
    Sector storage: sector_index 2 subspace_index_bra 1 subspace_index_ket 7 entries 110
  Input sector (raw labels): J 1 S 1 L 2 Lp 2 ipcut 61 dimension 120 mn 938.92 hw 20 ident 0
    After transposition to upper triangle: (Lp,L)=(2,2) flip 0
    Input sector properties: expected m.e. 7260 nmax 119 Nmax_bra 240 Nmax_ket 240
    Subspace labels: bra [ 2 1 1 0 0 ] (dimension 10) ket [ 2 1 1 0 0 ]  (dimension 10)
    Sector storage: sector_index 9 subspace_index_bra 7 subspace_index_ket 7 entries 110
  Input sector (raw labels): J 2 S 1 L 1 Lp 1 ipcut 61 dimension 119 mn 938.92 hw 20 ident 0
    After transposition to upper triangle: (Lp,L)=(1,1) flip 0
    Input sector properties: expected m.e. 7140 nmax 118 Nmax_bra 237 Nmax_ket 237
    Subspace labels: bra [ 1 1 2 1 1 ] (dimension 10) ket [ 1 1 2 1 1 ]  (dimension 10)
    Sector storage: sector_index 6 subspace_index_bra 5 subspace_index_ket 5 entries 110
  Input sector (raw labels): J 2 S 1 L 1 Lp 3 ipcut 61 dimension 119 mn 938.92 hw 20 ident 0
    After transposition to upper triangle: (Lp,L)=(1,3) flip 1
    Input sector properties: expected m.e. 7140 nmax 118 Nmax_bra 237 Nmax_ket 239
    Subspace labels: bra [ 1 1 2 1 1 ] (dimension 10) ket [ 3 1 2 1 1 ]  (dimension 9)
    Sector storage: sector_index 7 subspace_index_bra 5 subspace_index_ket 11 entries 90
  Input sector (raw labels): J 2 S 1 L 3 Lp 1 ipcut 61 dimension 119 mn 938.92 hw 20 ident 0
    After transposition to upper triangle: (Lp,L)=(1,3) flip 0
    Input sector properties: expected m.e. 7140 nmax 118 Nmax_bra 237 Nmax_ket 239
    Subspace labels: bra [ 1 1 2 1 1 ] (dimension 10) ket [ 3 1 2 1 1 ]  (dimension 9)
    Sector storage: sector_index 7 subspace_index_bra 5 subspace_index_ket 11 entries 90
  Input sector (raw labels): J 2 S 1 L 3 Lp 3 ipcut 61 dimension 119 mn 938.92 hw 20 ident 0
    After transposition to upper triangle: (Lp,L)=(3,3) flip 0
    Input sector properties: expected m.e. 7140 nmax 118 Nmax_bra 239 Nmax_ket 239
    Subspace labels: bra [ 3 1 2 1 1 ] (dimension 9) ket [ 3 1 2 1 1 ]  (dimension 9)
    Sector storage: sector_index 14 subspace_index_bra 11 subspace_index_ket 11 entries 90
  Input sector (raw labels): J 3 S 1 L 2 Lp 2 ipcut 61 dimension 119 mn 938.92 hw 20 ident 0
    After transposition to upper triangle: (Lp,L)=(2,2) flip 0
    Input sector properties: expected m.e. 7140 nmax 118 Nmax_bra 238 Nmax_ket 238
    Subspace labels: bra [ 2 1 3 0 0 ] (dimension 10) ket [ 2 1 3 0 0 ]  (dimension 10)
    Sector storage: sector_index 11 subspace_index_bra 9 subspace_index_ket 9 entries 110
  Input sector (raw labels): J 3 S 1 L 2 Lp 4 ipcut 61 dimension 119 mn 938.92 hw 20 ident 0
    After transposition to upper triangle: (Lp,L)=(2,4) flip 1
    Input sector properties: expected m.e. 7140 nmax 118 Nmax_bra 238 Nmax_ket 240
    Subspace labels: bra [ 2 1 3 0 0 ] (dimension 10) ket [ 4 1 3 0 0 ]  (dimension 9)
    Sector storage: sector_index 12 subspace_index_bra 9 subspace_index_ket 15 entries 90
  Input sector (raw labels): J 3 S 1 L 4 Lp 2 ipcut 61 dimension 119 mn 938.92 hw 20 ident 0
    After transposition to upper triangle: (Lp,L)=(2,4) flip 0
    Input sector properties: expected m.e. 7140 nmax 118 Nmax_bra 238 Nmax_ket 240
    Subspace labels: bra [ 2 1 3 0 0 ] (dimension 10) ket [ 4 1 3 0 0 ]  (dimension 9)
    Sector storage: sector_index 12 subspace_index_bra 9 subspace_index_ket 15 entries 90
  Input sector (raw labels): J 3 S 1 L 4 Lp 4 ipcut 61 dimension 119 mn 938.92 hw 20 ident 0
    After transposition to upper triangle: (Lp,L)=(4,4) flip 0
    Input sector properties: expected m.e. 7140 nmax 118 Nmax_bra 240 Nmax_ket 240
    Subspace labels: bra [ 4 1 3 0 0 ] (dimension 9) ket [ 4 1 3 0 0 ]  (dimension 9)
    Sector storage: sector_index 19 subspace_index_bra 15 subspace_index_ket 15 entries 90
  Input sector (raw labels): J 3 S 0 L 3 Lp 3 ipcut 61 dimension 118 mn 938.92 hw 20 ident 0
    After transposition to upper triangle: (Lp,L)=(3,3) flip 0
    Input sector properties: expected m.e. 7021 nmax 117 Nmax_bra 237 Nmax_ket 237
    Subspace labels: bra [ 3 0 3 0 1 ] (dimension 9) ket [ 3 0 3 0 1 ]  (dimension 9)
    Sector storage: sector_index 13 subspace_index_bra 10 subspace_index_ket 10 entries 90
  Input sector (raw labels): J 3 S 1 L 3 Lp 3 ipcut 61 dimension 118 mn 938.92 hw 20 ident 0
    After transposition to upper triangle: (Lp,L)=(3,3) flip 0
    Input sector properties: expected m.e. 7021 nmax 117 Nmax_bra 237 Nmax_ket 237
    Subspace labels: bra [ 3 1 3 1 1 ] (dimension 9) ket [ 3 1 3 1 1 ]  (dimension 9)
    Sector storage: sector_index 15 subspace_index_bra 12 subspace_index_ket 12 entries 90
  Input sector (raw labels): J 1 S 1 L 1 Lp 1 ipcut 61 dimension 119 mn 938.92 hw 20 ident 0
    After transposition to upper triangle: (Lp,L)=(1,1) flip 0
    Input sector properties: expected m.e. 7140 nmax 118 Nmax_bra 237 Nmax_ket 237
    Subspace labels: bra [ 1 1 1 1 1 ] (dimension 10) ket [ 1 1 1 1 1 ]  (dimension 10)
    Sector storage: sector_index 5 subspace_index_bra 4 subspace_index_ket 4 entries 110
  Input sector (raw labels): J 0 S 1 L 1 Lp 1 ipcut 61 dimension 119 mn 938.92 hw 20 ident 0
    After transposition to upper triangle: (Lp,L)=(1,1) flip 0
    Input sector properties: expected m.e. 7140 nmax 118 Nmax_bra 237 Nmax_ket 237
    Subspace labels: bra [ 1 1 0 1 1 ] (dimension 10) ket [ 1 1 0 1 1 ]  (dimension 10)
    Sector storage: sector_index 4 subspace_index_bra 3 subspace_index_ket 3 entries 110
  Input sector (raw labels): J 2 S 1 L 2 Lp 2 ipcut 61 dimension 119 mn 938.92 hw 20 ident 0
    After transposition to upper triangle: (Lp,L)=(2,2) flip 0
    Input sector properties: expected m.e. 7140 nmax 118 Nmax_bra 238 Nmax_ket 238
    Subspace labels: bra [ 2 1 2 0 0 ] (dimension 10) ket [ 2 1 2 0 0 ]  (dimension 10)
    Sector storage: sector_index 10 subspace_index_bra 8 subspace_index_ket 8 entries 110
  Input sector (raw labels): J 1 S 0 L 1 Lp 1 ipcut 61 dimension 119 mn 938.92 hw 20 ident 0
    After transposition to upper triangle: (Lp,L)=(1,1) flip 0
    Input sector properties: expected m.e. 7140 nmax 118 Nmax_bra 237 Nmax_ket 237
    Subspace labels: bra [ 1 0 1 0 1 ] (dimension 10) ket [ 1 0 1 0 1 ]  (dimension 10)
    Sector storage: sector_index 3 subspace_index_bra 2 subspace_index_ket 2 entries 110
  Input sector (raw labels): J 2 S 0 L 2 Lp 2 ipcut 61 dimension 119 mn 938.92 hw 20 ident 0
    After transposition to upper triangle: (Lp,L)=(2,2) flip 0
    Input sector properties: expected m.e. 7140 nmax 118 Nmax_bra 238 Nmax_ket 238
    Subspace labels: bra [ 2 0 2 1 0 ] (dimension 10) ket [ 2 0 2 1 0 ]  (dimension 10)
    Sector storage: sector_index 8 subspace_index_bra 6 subspace_index_ket 6 entries 110
  Input sector (raw labels): J 4 S 1 L 4 Lp 4 ipcut 61 dimension 118 mn 938.92 hw 20 ident 0
    After transposition to upper triangle: (Lp,L)=(4,4) flip 0
    Input sector properties: expected m.e. 7021 nmax 117 Nmax_bra 238 Nmax_ket 238
    Subspace labels: bra [ 4 1 4 0 0 ] (dimension 9) ket [ 4 1 4 0 0 ]  (dimension 9)
    Sector storage: sector_index 20 subspace_index_bra 16 subspace_index_ket 16 entries 90
  Input sector (raw labels): J 4 S 0 L 4 Lp 4 ipcut 61 dimension 118 mn 938.92 hw 20 ident 0
    After transposition to upper triangle: (Lp,L)=(4,4) flip 0
    Input sector properties: expected m.e. 7021 nmax 117 Nmax_bra 238 Nmax_ket 238
    Subspace labels: bra [ 4 0 4 1 0 ] (dimension 9) ket [ 4 0 4 1 0 ]  (dimension 9)
    Sector storage: sector_index 18 subspace_index_bra 14 subspace_index_ket 14 entries 90
  Input sector (raw labels): J 4 S 1 L 3 Lp 3 ipcut 61 dimension 118 mn 938.92 hw 20 ident 0
    After transposition to upper triangle: (Lp,L)=(3,3) flip 0
    Input sector properties: expected m.e. 7021 nmax 117 Nmax_bra 237 Nmax_ket 237
    Subspace labels: bra [ 3 1 4 1 1 ] (dimension 9) ket [ 3 1 4 1 1 ]  (dimension 9)
    Sector storage: sector_index 16 subspace_index_bra 13 subspace_index_ket 13 entries 90
  Input sector (raw labels): J 4 S 1 L 3 Lp 5 ipcut 61 dimension 118 mn 938.92 hw 20 ident 0
    After transposition to upper triangle: (Lp,L)=(3,5) flip 1
    Input sector properties: expected m.e. 7021 nmax 117 Nmax_bra 237 Nmax_ket 239
    Subspace labels: bra [ 3 1 4 1 1 ] (dimension 9) ket [ 5 1 4 1 1 ]  (dimension 8)
    Sector storage: sector_index 17 subspace_index_bra 13 subspace_index_ket 17 entries 72
  Input sector (raw labels): J 4 S 1 L 5 Lp 3 ipcut 61 dimension 118 mn 938.92 hw 20 ident 0
    After transposition to upper triangle: (Lp,L)=(3,5) flip 0
    Input sector properties: expected m.e. 7021 nmax 117 Nmax_bra 237 Nmax_ket 239
    Subspace labels: bra [ 3 1 4 1 1 ] (dimension 9) ket [ 5 1 4 1 1 ]  (dimension 8)
    Sector storage: sector_index 17 subspace_index_bra 13 subspace_index_ket 17 entries 72
  Input sector (raw labels): J 4 S 1 L 5 Lp 5 ipcut 61 dimension 118 mn 938.92 hw 20 ident 0
    After transposition to upper triangle: (Lp,L)=(5,5) flip 0
    Input sector properties: expected m.e. 7021 nmax 117 Nmax_bra 239 Nmax_ket 239
    Subspace labels: bra [ 5 1 4 1 1 ] (dimension 8) ket [ 5 1 4 1 1 ]  (dimension 8)
    Sector storage: sector_index 21 subspace_index_bra 17 subspace_index_ket 17 entries 72
  Reached termination marker.
Writing relative operator file...
  Filename: jisp16_Nmax20_hw20.0_rel.dat
  Operator properties: J0 0 g0 0 T0_min 0 T0_max 0 symmetry 0
  Truncation: Nmax 20 Jmax 4
